drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,18 +32,112 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #error DTCM for DMA buffer is activated but zephyr,dtcm is not present in dts
 #endif
 
+static HAL_StatusTypeDef ETH_RD_PHY_REG( ETH_HandleTypeDef* ps_heth,
+                                         uint32_t u32_PHYAddr,
+                                         uint32_t u32_PHYReg,
+                                         uint32_t* pu32_RegVal);
+
+#ifndef PHY_ADDR
+#define PHY_ADDR        ((uint32_t)CONFIG_ETH_STM32_HAL_PHY_ADDRESS)
+#endif
+
+#ifndef PHY_BSR
+#define PHY_BSR     ((uint16_t)0x0001U)
+#endif
+
+#ifndef PHY_LINKED_STATUS
```

Comment:
Why checking `ifndef`?

Reply:
> Why checking `ifndef`?

In case of PHY_BSR defined somewhere else. Should `#ifndef` be deleted?